model/listeners.py

```diff
@@ -74,9 +80,9 @@ def _site_configuration_has_changed(_db, cooldown=1):
             _db = Session.object_session(_db)
 
         # Update the timestamp.
-        now = datetime.datetime.utcnow()
+        now = utc_now()
         earlier = now-datetime.timedelta(seconds=cooldown)
-        sql = "UPDATE timestamps SET finish=:finish WHERE service=:service AND collection_id IS NULL AND finish<=:earlier;"
+        sql = "UPDATE timestamps SET finish=(:finish at time zone 'utc') WHERE service=:service AND collection_id IS NULL AND finish<=(:earlier at time zone 'utc');"
```

Comment:
This turned out to be a big problem.

When using this branch, I created a library "library1" successfully but when I went to `:6500/library1/groups`, the page had an internal server error. After lots of digging around, in `reload_settings_if_changed` the time check was off and `self.load_settings()` was never called. This meant that the app didn't know about this new library even though it was successfully created and was in the database.